lib/src/characters_impl.dart

```diff
@@ -464,15 +464,15 @@ class StringCharacterRange implements CharacterRange {
 
   /// Creates a [Breaks] from [_end] to `_string.length`.
   ///
-  /// Uses information stored in [_state] for cases where the next
-  /// character has already been seen.
+  /// Uses information stored in the state for cases where the next character
+  /// has already been seen.
```

Comment:
I'm not even sure which state that is. Maybe the entire paragraph is garbage.

Reply:
😁